net-irc/irssi/irssi-1.1.1.recipe

```diff
@@ -4,14 +4,14 @@ It uses a text-mode user interface. Numerous Perl scripts have been \
 made available for Irssi to customise how it looks and operates. Plugins \
 are available which add encryption and protocols such as ICQ and XMPP."
 HOMEPAGE="https://irssi.org/"
-COPYRIGHT="1999-2016 Timo Sirainen"
+COPYRIGHT="1999-2018 Timo Sirainen"
```

Comment:
There're more people than just Timo developing irssi. This should be updated to include all copyright holders.